core/src/main/java/org/jboss/pnc/build/finder/protobuf/MultiValuedMapProtobufWrapper.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.build.finder.protobuf;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.collections4.MultiValuedMap;
+import org.apache.commons.collections4.multimap.HashSetValuedHashMap;
+import org.infinispan.protostream.WrappedMessage;
+import org.infinispan.protostream.annotations.ProtoFactory;
+import org.infinispan.protostream.annotations.ProtoField;
+
+/**
+ * Class wrapping around MultiValuedMap interface (via HashSetValuedHashMap) so that it can be marshalled with Protobuf
+ *
+ * @param <K> The key type
+ * @param <V> The value type
+ */
+public class MultiValuedMapProtobufWrapper<K, V> extends HashSetValuedHashMap<K, V> {
+    private static final long serialVersionUID = 20211111L;
```

Comment:
This value doesn't look "right". There's a way to generate the real value as generated by `serialver` in an IDE, or call it manually.